packages/monorepo-scripts/src/publish.ts

```
@@ -38,8 +38,14 @@ const packageNameToWebsitePath: { [name: string]: string } = {
 };
 
 (async () => {
+    const hasRequiredSetup = await checkPublishRequiredSetupAsync();
+    if (!hasRequiredSetup) {
+        return; // abort
```

Wouldn't it be better to throw a helpfull error message then to just silently fail?

---

We print helpful messages for each conditions within `checkPublishRequiredSetupAsync`. Do you still think this is necessary?